### ISSUE TITLE
Add test

### ISSUE THREAD

**Issue by ARandomOWL** [tuura/boolean#4]

**Comment by snowleopard** [comment]
Yes, it would be great to add some testing. I made a commit but there is no way to check it makes any sense apart from type-checking with GHC :)

**Comment by ARandomOWL** [comment]
I will add some basic tests like those in tuura/plato in the near future.
